lib/commons/color/get-background-color.js

```diff
@@ -65,9 +77,48 @@ function calculateObscuringAlpha(elmIndex, elmStack) {
 }
 
 function elmOutsideParent(elm, bgElm, bgColor) {
```

Comment:
@marcysutton I think we should do is lift that `bgColor.alpha !== 0` check out of this function. That would make more sense for it.

The purpose of this check is to allow us to drop an element into cantTell if the parent giving it a background only partially overlaps the child element. E.g. that element may have multiple background colors, which we're not set up to test.